app/components/connection_banner/connection_banner.tsx

```diff
@@ -109,7 +109,7 @@ const ConnectionBanner = ({
             clearTimeoutRef(openTimeout);
             clearTimeoutRef(closeTimeout);
         };
-    }, []);
+    }, []); // eslint-disable-line react-hooks/exhaustive-deps -- only run on mount
```

Comment:
Is this just to suppress the warning? I know in the web app we don't need to do this usually.